KebabBackend/routes/api.php

```diff
@@ -3,93 +3,74 @@
-
         Route::delete('{comment}', [CommentController::class, 'removeComment']);
     });
 });
 
 Route::prefix('admin')->middleware(['auth:sanctum', 'admin'])->group(function () {
-
     Route::get('users', [UserController::class, 'index']);
-
     Route::get('users/{id}', [UserController::class, 'getUserById']);
-
     Route::delete('delete-user/{id}', [UserController::class, 'destroy']);
-
     Route::post('change-password-first-login', [UserController::class, 'changePasswordForFirstLogin']);
-
     Route::delete('delete-comment/{comment}', [CommentController::class, 'adminRemoveComment']);
 });
 
 Route::prefix('kebabs')->group(function () {
-
     Route::get('{kebab}', [KebabController::class, 'show']);
-
     Route::get('/', [KebabController::class, 'index']);
-
     Route::get('{kebab}/comments', [CommentController::class, 'getCommentsByKebabId']);
-
     Route::middleware(['auth:sanctum'])->post('{kebab}/comments', [CommentController::class, 'addComment']);
-
     Route::middleware(['auth:sanctum'])->post('{kebab}/favourite', [FavouriteController::class, 'addToFavourites']);
-
     Route::middleware(['auth:sanctum'])->delete('{kebab}/favourite', [FavouriteController::class, 'removeFromFavourites']);
-
     Route::middleware(['auth:sanctum', 'admin'])->post('/', [KebabController::class, 'store']);
-
     Route::middleware(['auth:sanctum', 'admin'])->put('{kebab}', [KebabController::class, 'update']);
-
     Route::middleware(['auth:sanctum', 'admin'])->delete('{kebab}', [KebabController::class, 'destroy']);
 });
 
 Route::prefix('meattypes')->group(function () {
     Route::get('/', [MeatTypeController::class, 'index']);
-
     Route::middleware(['auth:sanctum', 'admin'])->post('/', [MeatTypeController::class, 'store']);
-
     Route::middleware(['auth:sanctum', 'admin'])->put('{id}', [MeatTypeController::class, 'update']);
-
     Route::middleware(['auth:sanctum', 'admin'])->delete('{id}', [MeatTypeController::class, 'destroy']);
 });
 
 Route::prefix('saucetypes')->group(function () {
     Route::get('/', [SauceTypeController::class, 'index']);
-
     Route::middleware(['auth:sanctum', 'admin'])->post('/', [SauceTypeController::class, 'store']);
-
     Route::middleware(['auth:sanctum', 'admin'])->put('{id}', [SauceTypeController::class, 'update']);
-
     Route::middleware(['auth:sanctum', 'admin'])->delete('{id}', [SauceTypeController::class, 'destroy']);
 });
 
+Route::middleware('auth:sanctum')->group(function () {
+    Route::post('reports', [ReportController::class, 'store']);
+});
+
+Route::middleware(['auth:sanctum', 'admin'])->prefix('admin/reports')->group(function () {
+    Route::get('/', [ReportController::class, 'index']);
+    Route::delete('{report}', [ReportController::class, 'destroy']);
+    Route::put('{report}/accept', [ReportController::class, 'acceptReport']);
+    Route::put('{report}/refuse', [ReportController::class, 'refuseReport']);
+});
```

Comment:
These paths are not represented correctly in the docs, it would be beneficial to adjust the docs to fit these paths